testing/postgres-client-tests/node/workbenchTests/diffs.js

```diff
@@ -248,8 +389,22 @@ export const diffTests = [
           diff_type: "schema",
           statement: "DROP TABLE `test_info`;",
         },
+        // TODO: Should `CREATE SCHEMA` statement be included here?
```

Comment:
Yes, please record with an issue

Reply:
Issue for both comments here https://github.com/dolthub/doltgresql/issues/796